kubevirtmng/ui/DiskList.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import $ from "jquery"
+
+
+class Volume extends React.Component {
+  constructor() {
```

Comment:
It's better to use:

constructor(props) {
  super(props)
  ...

Just in case Component ever stops setting props itself (after object instantiation). super(props) is a reasonable safeguard, which matches the official docs